src/NzbDrone.Core/Localization/Core/cs.json

```diff
@@ -5,21 +5,21 @@
     "AgeWhenGrabbed": "Stáří (kdy bylo získáno)",
     "Always": "Vždy",
     "AnalyseVideoFiles": "Analyzovat video soubory",
-    "AnalyseVideoFilesHelpText": "Extrahujte ze souborů informace o videu, jako je rozlišení, doba běhu a informace o kodeku. To vyžaduje, aby Sonarr četl části souboru, což může způsobit vysokou aktivitu disku nebo sítě během skenování.",
+    "AnalyseVideoFilesHelpText": "Extrahujte ze souborů informace o videu, jako je rozlišení, doba běhu a informace o kodeku. To vyžaduje, aby {appName} četl části souboru, což může způsobit vysokou aktivitu disku nebo sítě během skenování.",
     "ApplicationURL": "URL aplikace",
     "ApplicationUrlHelpText": "Externí adresa URL této aplikace včetně http(s)://, portu a základní adresy URL",
-    "AuthenticationMethodHelpText": "Vyžadovat uživatelské jméno a heslo pro přístup k Sonarru",
+    "AuthenticationMethodHelpText": "Vyžadovat uživatelské jméno a heslo pro přístup k {appName}u",
```

Comment:
/bSonarr/

Reply:
Nah, need to do all languages to fix things or they may never get changed.